app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -167,6 +169,10 @@ protected void onCreate(final Bundle savedInstanceState) {
             // if this is enabled by the user.
             NotificationWorker.initialize(this);
         }
+        if (!UpdateSettingsFragment.wasUserAskedForConsent(this)
+                && ReleaseVersionUtil.INSTANCE.isReleaseApk()) {
+            UpdateSettingsFragment.askForConsentToUpdateChecks(this);
+        }
```

Comment:
Mmmh, I'm not sure about this change. It will be shown to all users when the app is started the first time, right? Wouldn't it be a not-so-welcoming first screen?

I would prefer if it were shown the _second_ time the app is started, like this:
```suggestion
        if (!UpdateSettingsFragment.wasUserAskedForConsent(this)
                && ReleaseVersionUtil.INSTANCE.isReleaseApk()
                && !App.getApp().isFirstRun()) {
            UpdateSettingsFragment.askForConsentToUpdateChecks(this);
        }
```